src/transaction_builder.js

```diff
@@ -150,6 +153,16 @@ export class TransactionBuilder {
     return this;
   }
 
+  /**
+   * Adds a memo to the transaction.
```

Comment:
@Akuukis I think the description here is incorrect? Can you update it to reflect what it does?